### ISSUE TITLE
⊕ at start of a row is larger than normal

### ISSUE THREAD

**Issue by hbghlyj** [ronkok/Temml#29]
When converting
```
\begin{align}
\oplus\oplus\\
\oplus\oplus
\end{align}
```
![image](https://github.com/ronkok/Temml/assets/53823634/da0f0fb6-d1be-4c9c-81ba-1194ccb1c237)
to MathML, ⊕ at start of a row appears larger than normal.

**Browser:** Firefox/110.0

Same result using
```
\begin{align}
⊕⊕\\⊕⊕
\end{align}
```
![image](https://github.com/ronkok/Temml/assets/53823634/5b894cb1-0c04-4903-b39b-a452ead6f3ed)

**Comment by ronkok** [comment]
Firefox is treating the first `⊕` as an emoji. That's annoying.

I'll work up a fix so that Temml emits a `&#xFE0E;` after a `⊕`. That will prevent the emoji.

It may be a few days before I get to this, but it will be fixed this week.

**Comment by ronkok** [comment]
This issue is fixed by release 0.10.16.
